build-system/common/check-package-manager.js

```diff
@@ -183,9 +183,7 @@ function getNodeLatestLtsVersion(distributionsJson) {
 // If yarn is being run, perform a version check and proceed with the install.
 function checkYarnVersion() {
   const yarnVersion = getStdout(yarnExecutable + ' --version').trim();
-  const yarnInfo = getStdout(yarnExecutable + ' info --json yarn').trim();
-  const yarnInfoJson = JSON.parse(yarnInfo.split('\n')[0]); // First line
-  const stableVersion = getYarnStableVersion(yarnInfoJson);
+  const stableVersion = '1.22.4';
```

Comment:
Why not pin to 1.22.5, which is the latest stable version before 1.22.10? It appears to have the same signature as 1.22.4. Here's what I'm seeing:

<img width="1056" alt="Screen Shot 2020-10-01 at 1 23 20 PM" src="https://user-images.githubusercontent.com/26553114/94842496-762f3280-03e9-11eb-89ad-0e075510bc34.png">

And a higher level follow up question: If `1.22.5` is indeed a good version, do we need to pin the version at all?

Reply:
Pinning to `1.22.5` makes sense as well. The last good version was the intent, not a specific version number.

Why pin? To avoid attempting to install an invalid version (`1.22.10` at the current time). This PR is intended to be reverted once `yarn` is back to working as expected with the latest stable release.

Reply:
Updated to pin to 1.22.5.